src/ui/auth/rackspace/rackspace.go

```diff
@@ -161,10 +168,33 @@ func (l *Auth) Authenticate(m models.AuthModel) (*models.User, error) {
 }
 
 func init() {
+	log.Infof("Initialing Rackspace Managed Kubernetes Auth")
+
 	auth.Register("rackspace_mk8s_auth", &Auth{})
 	rackspaceMK8SAuthURL := config.RackspaceMK8SAuthURL()
 	rackspaceMK8SAuthURLTokenEndpoint = rackspaceMK8SAuthURL + "/authenticate/token"
-	log.Infof("Initializing Rackspace Managed Kubernetes Auth: rackspaceMK8SAuthURL=%s", rackspaceMK8SAuthURL)
+
+	if strings.HasPrefix(rackspaceMK8SAuthURL, "https") {
```

Comment:
nit: I prefer to avoid placing stuff like this in `init` because it makes it hard to test. but it's fine here